app/Jobs/ProcessSurfacePressureJob.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Models\HistoricalSurfacePressureProcessingReportsModel;
+use App\Service\Surface_Pressure\ProcessedSurfacePressureService;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+
+class ProcessSurfacePressureJob implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    private HistoricalSurfacePressureProcessingReportsModel $historicalSurfacePressureProcessingRepostsModel;
```

Comment:
```suggestion
    private HistoricalSurfacePressureProcessingReportsModel $historicalSurfacePressureProcessingReportsModel;
```
typo, Reports, not RepoSts, this isn't reddit :D